llvm/include/llvm/IR/VectorTypeUtils.h

```diff
@@ -14,6 +14,11 @@
 
 namespace llvm {
 
+/// Identifies if the vector form of the intrinsic that returns a struct has
+/// a scalar element at the struct element index \p EleIdx.
+LLVM_ABI bool isVectorIntrinsicWithStructReturnScalarAtField(unsigned ID,
```

Comment:
```suggestion
LLVM_ABI bool isVectorIntrinsicWithStructReturnScalarAtField(unsigned IID,
```
for consistency with change sblow